services/minespace-web/src/components/Forms/tailing/tailingsStorageFacility/EngineerOfRecord.js

```diff
@@ -151,6 +188,25 @@ export const EngineerOfRecord = (props) => {
             type="info"
           />
         )}
+
+        {isNumber(daysToEORExpiry) && daysToEORExpiry >= 0 && daysToEORExpiry <= 30 && (
+          <Alert
+            message="Engineer of Record will Expire within 30 Days"
+            description="To be in compliance, you must have a current, Ministry-approved Engineer of Record on file."
+            showIcon
+            type="warning"
+          />
+        )}
+
+        {isNumber(daysToEORExpiry) && daysToEORExpiry < 0 && (
```

Comment:
Would it make sense to add an `|| !daysToEORExpiry`?  Or is that covered by the submission validation?

Reply:
I think the intention with this warning was just to be an extra warning in case that the current EOR has expired. If you have yet to assign an EoR you will get the softer "There's no EOR on file" warning, and can't continue in the flow without assigning one

Reply:
:+1: